devel/asio/Portfile

```diff
@@ -2,26 +2,30 @@
 
 PortSystem                  1.0
 PortGroup                   openssl 1.0
+PortGroup                   muniversal 1.0
 
 name                        asio
-version                     1.20.0
+version                     1.22.1
 revision                    0
-checksums                   rmd160  762d0c9dec18b7ce0fdf9069067900ab1345887f \
-                            sha256  204374d3cadff1b57a63f4c343cbadcee28374c072dc04b549d772dbba9f650c \
-                            size    1845826
+checksums           rmd160  02ed50e944d1ebe835fa15d0e90dfd46c974ecf5 \
+                    sha256  6874d81a863d800ee53456b1cafcdd1abf38bbbf54ecf295056b053c0d7115ce \
+                    size    3085878
 
 license                     Boost-1
 description                 Asio C++ Library.
 long_description            Asio is a cross-platform C++ library for network and low-level I/O \
-                            programming that provides developers with a \
-                            consistent asynchronous model using a modern C++ approach.
+                            programming that provides developers \
+                            with a consistent asynchronous model using a modern C++ approach.
```

Comment:
please do not make unneeded whitespace changes, they clutter the diffs

Reply:
These small things will never get fixed if no one ever bothers to do a one-time change.

Reply:
I fail to see what problem you are "fixing" here

Reply:
It is not a compilation problem of course, but a bad typography. (Well, of course if there are good reasons for having it this way, I won’t insist.)

Reply:
I don't see anything wrong with it, both for content nor style - please leave as-is.

Reply:
@barracuda156 please revert this

Reply:
> I don't see anything wrong with it, both for content nor style - please leave as-is.

How about if I make this correction in a separate commit? (It _is_ wrong – articles and prepositions should not be “hanging” in the end of a line. I worked 10+ years in desktop publishing in academia, I know what I am talking about here.)